### PR TITLE
Login screen text

### PR DESCRIPTION
**Pull request by mustiwebp**
New Login screen has been implemented.

### CODE REVIEW

**Comment by S1rDan** [comment]
You have to update the assets for this one as well.

File: lib/screens/login_screen.dart [hunk 1]
```diff
@@ -95,60 +88,90 @@ class _LoginScreenState extends State<LoginScreen>
...
-                    SizedBox(
-                      width: 200,
-                      child: TextField(
-                        controller: _loginController,
-                        decoration: const InputDecoration(
-                          border: OutlineInputBorder(),
-                          labelText: 'Login',
-                        ),
-                      ),
+        child: SingleChildScrollView(
```
Comment (S1rDan) [inline lib/screens/login_screen.dart:91]: I would suggest, you use PicosBody and PicosScreenFrame, if nothing speaks against it.

File: lib/screens/login_screen.dart [hunk 2]
```diff
@@ -95,60 +88,90 @@ class _LoginScreenState extends State<LoginScreen>
...
                       ),
-                    ),
-                    ElevatedButton(
-                      onPressed: () => _submitHandler(
-                        _loginController.text,
-                        _passwordController.text,
-                        context,
+                      TextSpan(
+                        text:
+                            '''Vielen Dank, dass Sie sich für die Teilnahme an dem Projekt DISTANCE entschieden haben. Wir wünschen viel Freude bei der Nutzung der PICOS App!''',
```
Comment (S1rDan) [inline lib/screens/login_screen.dart:114]: Please use the localization.

**Review by S1rDan — CHANGES_REQUESTED** [review]

File: lib/screens/login_screen.dart [hunk 3]
```diff
@@ -92,63 +88,93 @@ class _LoginScreenState extends State<LoginScreen>
...
+                const SizedBox(
+                  height: 15,
+                ),
+                SizedBox(
+                  width: 200,
+                  child: TextField(
+                    controller: _loginController,
+                    decoration: InputDecoration(
+                      border: const OutlineInputBorder(),
+                      labelText: AppLocalizations.of(context)!.login,
```
Comment (S1rDan) [inline lib/screens/login_screen.dart:134]: You are using "Einloggen" here and it's wrong in this context. What you want to say is "Login".

File: lib/screens/login_screen.dart [hunk 4]
```diff
@@ -92,63 +88,93 @@ class _LoginScreenState extends State<LoginScreen>
...
+                ),
+                SizedBox(
+                  width: 200,
+                  child: PicosInkWellButton(
+                    onTap: () => _submitHandler(
+                      _loginController.text,
+                      _passwordController.text,
+                      context,
+                    ),
+                    text: AppLocalizations.of(context)!.login,
```
Comment (S1rDan) [inline lib/screens/login_screen.dart:158]: I would also suggest you use here "Submit" / "Senden", as in the Mockup depicted.

File: lib/screens/login_screen.dart [hunk 5]
```diff
@@ -92,63 +88,93 @@ class _LoginScreenState extends State<LoginScreen>
...
+                  ),
+                ),
+                _loginfailure
+                    ? Text(AppLocalizations.of(context)!.wrongPassword)
```
Comment (S1rDan) [inline lib/screens/login_screen.dart:162]: Don't tell the user, if the password is wrong. Tell him that the credentials are wrong (more general).

File: lib/screens/login_screen.dart [hunk 6]
```diff
@@ -92,63 +88,93 @@ class _LoginScreenState extends State<LoginScreen>
...
-                      child: const Text('Submit'),
+                    ],
+                  ),
+                ),
+                const SizedBox(
+                  height: 15,
+                ),
+                SizedBox(
+                  width: 200,
+                  child: TextField(
```
Comment (S1rDan) [inline lib/screens/login_screen.dart:130]: I would suggest using PicosTextField, if you haven't tried it yet.
Same for the password field.

File: lib/screens/login_screen.dart [hunk 7]
```diff
@@ -92,63 +88,93 @@ class _LoginScreenState extends State<LoginScreen>
...
-                      child: TextField(
-                        controller: _loginController,
-                        decoration: const InputDecoration(
-                          border: OutlineInputBorder(),
-                          labelText: 'Login',
-                        ),
-                      ),
+        child: PicosBody(
+          padding: 0,
+          child: Padding(
```
Comment (S1rDan) [inline lib/screens/login_screen.dart:95]: Why do you use Padding, if you want to remove Padding from PicosBody?

File: lib/widgets/picos_body.dart [hunk 8]
```diff
@@ -20,15 +20,22 @@ import 'package:flutter/material.dart';
...
   /// Creates the PicosBody.
-  const PicosBody({required this.child, Key? key}) : super(key: key);
+  const PicosBody({
+    required this.child,
+    this.padding = 10,
+    Key? key,
+  }) : super(key: key);
+
+  /// Padding value of the main widget.
+  final double padding;
```
Comment (S1rDan) [inline lib/widgets/picos_body.dart:30]: These changes should may be rethought.

**Review by S1rDan — CHANGES_REQUESTED** [review]

File: lib/widgets/picos_body.dart (hunk 8 above)
Comment (mustiwebp) [reply lib/widgets/picos_body.dart:30]: The changes have been redone.

File: lib/screens/login_screen.dart (hunk 7 above)
Comment (mustiwebp) [reply lib/screens/login_screen.dart:95]: Valid Objection, redone!

**Review by S1rDan — APPROVED** [review]
Good Job,
just fix the typo and we are good to go!

**Pull request merged**